contracts/StrategySushiswapPair.sol

```diff
@@ -173,24 +174,10 @@ contract StrategySushiswapPair is BaseStrategy {
      * Liquidate as many assets as possible to `want`, irregardless of slippage,
      * up to `_amount`. Any excess should be re-invested here as well.
      */
-    function liquidatePosition(uint256 _amount) internal override {
-        // TODO: Do stuff here to free up `_amount` from all positions back into `want`
+    function liquidatePosition(uint256 _amount) internal override returns (uint256 _amountFreed) {
+        uint256 before = want.balanceOf(address(this));
         SushiChef(chef).withdraw(pid, _amount);
```

Comment:
I think `_amount` might be greater than what you have available.

Reply:
nope! Vault.py was recently changed:
https://github.com/iearn-finance/yearn-vaults/blob/master/contracts/Vault.vy#L751

Reply:
But you are withdrawing an amount above your current balance. Why? Surely you want to withdraw _amount - before.

Reply:
Yeah I think you are right @Grandthrax